infrastructure/variables.tf

```diff
@@ -53,10 +53,10 @@ variable "running_in_cloud" {
 
 # This is a placeholder until there is a production image ready.
 variable "workers_docker_image" {
-  default = "wkurt/dr_worker:3"
+  default = "wkurt/dr_worker:5"
```

Comment:
Why can't we use `ccdl/data_refinery_workers` and `ccdl/data_refinery_foreman` images?

Reply:
Are those images ready to go? I definitely can switch to them, I just wasn't sure what their status was and was trying to keep this PR's scope small.

Reply:
Yes, circleci deployment always updates these two images after `data_refinery_worker_base`.